workflows-fiji.html

```diff
@@ -405,11 +405,11 @@
 
 <p><span class="bold">Line 1:</span></span><br/>
 From:&nbsp;&nbsp;&nbsp;<span class="code-example">USERNAME = &#60;OMERO USERNAME&#62;</span><br/>
-To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;USERNAME = &#60;user-x&#62;</span></p>
+To:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;USERNAME = &#60;your_username&#62;</span></p>
```

Comment:
These examples only use code example markup for the text you're replacing but all the examples above use it for the existing text and the replacement text. It would be nice to have this consistent throughout the section and I favour having the earlier examples follow this style and use different fonts.